projects/npm-tools/packages/changelog-generator/src/yarn.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const child_process = require('child_process');
+const fs = require('fs');
+
+const {error, info} = require('./console');
+
+const pjson = JSON.parse(fs.readFileSync('package.json'));
```

Comment:
We don't use abbreviatures, so `packageJson` (or sometimes, I use `pkgJson`). But in general we don't like abbreviatures (I think there's something on that on the guidelines...).

Reply:
Changed and pushed, thanks for your feedback @izaera!